binder-index.md

```diff
@@ -78,15 +78,15 @@ These are noted in the README.md files for each sample, along with complete inst
       <td><strong>Algorithms:</strong></td>
       <td><a href="./samples/algorithms/chsh-game/README.md"><strong>CHSH Game</strong></a></td>
       <td></td>
-      <td><a href="./samples/algorithms/chsh-game/host.py">Q# + Python</a></td>
```

Comment:
There's a separate column for Q# standalone, which generally doesn't have links since those don't show up nicely in mybinder.org for plain Q# files. Could you please remove these two links entirely and add "Q# standalone" to the appropriate columns? Thanks!

Reply:
Absolutely! Just pushed the change